src/platform/Darwin/DiagnosticDataProviderImpl.cpp

```diff
@@ -89,6 +89,21 @@ CHIP_ERROR DiagnosticDataProviderImpl::ResetWatermarks()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR DiagnosticDataProviderImpl::GetRebootCount(uint16_t & rebootCount)
+{
+    uint32_t count = 0;
+
+    CHIP_ERROR err = ConfigurationMgr().GetRebootCount(count);
+
+    if (err == CHIP_NO_ERROR)
```

Comment:
How about the below to reduce err and {}
ReturnErrorOnFailure(ConfigurationMgr().GetRebootCount(count));
VerifyOrReturnError(count <= UINT16_MAX, CHIP_ERROR_INVALID_INTEGER_VALUE);
rebootCount = static_cast<uint16_t>(count);
return CHIP_NO_ERROR;

Reply:
Done.